NeuroID.xcodeproj/project.pbxproj

```diff
@@ -1094,8 +1123,9 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 153C967454B5320D895ADE13 /* Pods-SDKTest.debug.xcconfig */;
 			buildSettings = {
+				CODE_SIGN_IDENTITY = "Apple Distribution: NEURO-ID, INC. (9B7FQ8D7B8)";
```

Comment:
Is this housed in fast lane and s3 or created on your local machine or?

Reply:
All in fastlane/s3. https://s3.console.aws.amazon.com/s3/buckets/fastlane20230807215426085400000001?region=us-east-1&bucketType=general&prefix=9B7FQ8D7B8/&showversions=false